src/projectile.cpp

```diff
@@ -200,6 +202,19 @@ void apply_ammo_effects( const tripoint &p, const std::set<std::string> &effects
             }
         }
     }
+
+    if( effects.count( "OVERPRESSURE_WAVE" ) > 0 ) {
+        // add_msg( m_bad, _( "A buzzing wave of turbulence bursts forth!" ) );
+        std::vector<point> sight = line_to( g->u.posx, g->u.posy, p->posx(), p->posy(), 0 );
```

Comment:
`p` isn't a pointer type so can't use arrow operator there
```suggestion
        std::vector<point> sight = line_to( g->u.posx, g->u.posy, p.posx(), p.posy(), 0 );
```

Reply:
Done